Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -436,10 +447,8 @@ function Set-SQLServerRequiredTLSVersion
     if(-not($AutoRemediation))
     {
         Write-Host "Following SQL Servers are having TLS version either not set or less than required minimal TLS version less than required TLS Version:" -ForegroundColor $([Constants]::MessageType.Info)
-        $colsProperty =     @{Expression={$_.ServerName};Label="Server Name";Width=10;Alignment="left"},
```

Comment:
Any reason we had "Server Name" and "Location" being printed before?

Reply:
For this. BRS, information about server version and location provides no value. That's why removed it.

Earlier, it may have been copied from other SQL Server BRS where it may have been of some use about which I am not aware of.